src/tools/genpolicy/rules.rego

```diff
@@ -1329,6 +1372,9 @@ ExecProcessRequest {
 
     regex.match(p_regex, i_command)
 
+    some p_container in policy_data.containers
```

Comment:
same as in https://github.com/kata-containers/kata-containers/pull/11233/commits/12c03ffcf64f8d82122acab060a884e8579a60bd#r2074144387